client/ladder/ladder.tsx

```diff
@@ -162,6 +172,35 @@ export function Ladder({ matchmakingType: routeType }: LadderProps) {
     }
   }, [routeType])
 
+  let content = <LoadingDotsArea />
+  if (searchQuery && searchResults) {
+    content = (
+      <LadderTable
```

Comment:
imo, given how many props this has, you'd be better off just setting an object to pull `lastUpdated`, `totalCount`, and `players` from rather than setting an entire element. How you have it now is extremely prone to causing problems with future updates